.github/workflows/build-mesa.yaml

```diff
@@ -0,0 +1,34 @@
+name: "Build OpenGL/Vulkan CTS"
+
+on:
+  workflow_dispatch:
+    inputs:
+      arch:
+        description: 'Runner to run the workflow'
+        required: true
+        default: 'ubuntu-24.04'
+        type: choice
+        options:
+          - ubuntu-24.04
+          - ubuntu-24.04-arm
```

Comment:
Interesting, I thought the correct tag for arm64 machines among our runners is "ARM64", haven't found any tagged "ubuntu-24.04-arm" so far

Reply:
This is the tags for the public github runner. (Available for free for any public GH project)

Reply:
I see, if you put the "self-hosted, noble, ARM64" it should be picked up by canonical runners, see [my job](https://github.com/canonical/checkbox-gfx-asa/actions/runs/15158398911/job/42618569245)